build_pip_pkg.sh

```diff
@@ -59,7 +59,7 @@ function main() {
   
   python setup.py bdist_wheel
 
-  cp dist/*.whl "${DEST}"
+  auditwheel repair -w "${DEST}" dist/*.whl
```

Comment:
We should probably note this at the beginning of the script as a requirement since most users might not have this installed.